cijoe/configs/perf-lat-freebsd.toml

```diff
@@ -0,0 +1,71 @@
+[transport.ssh]
+hostname = "perf-lat-fbsd.home.arpa"
+port = 22
+username = "root"
+password = "root"
+
+[os]
+name = "freebsd"
+version = "13"
+
+[xnvme.repository]
+remote = "https://github.com/OpenMPDK/xNVMe.git"
```

Comment:
Replace the old org. link:
https://github.com/OpenMPDK/xNVMe.git
with
https://github.com/xnvme/xnvme.git